File: sdk/typescript/examples/04_kv_basic.mjs
// 04_kv_basic:KV set / get / delete。
import { client } from "./00_helpers.mjs";
const combee = client();
const cell = await combee.cells.create();
await cell.kv.set("greeting", "hello");
console.log("greeting:", await cell.kv.get("greeting"));
await cell.kv.delete("greeting");
console.log("after delete:", await cell.kv.get("greeting"));

File: sdk/python/examples/04_kv_basic.py
"""04_kv_basic:KV set / get / delete。"""
from helpers import client

combee = client()
cell = combee.cells.create()
cell.kv.set("greeting", "hello")
print("greeting:", cell.kv.get("greeting"))
cell.kv.delete("greeting")
print("after delete:", cell.kv.get("greeting"))
